### ISSUE TITLE
Score exported to MEI and reimported gives error and doesn't recreate the original

### ISSUE THREAD

**Issue by retorquere** [musescore/MuseScore#22519]
### Issue type

Opening/saving issue

### Bug description

Score exported to MEI and reimported gives error and doesn't recreate the original

### Steps to reproduce

* open attached mscx file
* save as .mei
* open .mei

### Screenshots/Screen recordings

<img width="948" alt="after" src="https://github.com/musescore/MuseScore/assets/132108/891de72e-d670-4167-b5c1-d7753b2774d3">
<img width="521" alt="error" src="https://github.com/musescore/MuseScore/assets/132108/a66e9bb4-aa4e-4be7-99aa-59b6fc70e016">

### MuseScore Version

4.2.1.240230937

### Regression

I don't know

### Operating system

MacOS 14.4.1

### Additional context

[Zipped .mscz to allow uploading](https://github.com/musescore/MuseScore/files/15061405/MoneyBeat.zip)

**Comment by cbjeukendrup** [comment]
@lpugin or @rettinghaus Could you please take a look at this?

**Comment by rettinghaus** [comment]
Unpitched music is not supported yet. Neither are different head shapes. 

(For the latter we need to adjust the MEI customization first.)

**Comment by Jojo-Schmitz** [comment]
That'd make this a feature request, right?

**Comment by rettinghaus** [comment]
Yes, but more of a feature request for [MEI Basic](https://github.com/music-encoding/music-encoding), because it is not yet possible to encode unpitched music nor special head shapes.

**Comment by Jojo-Schmitz** [comment]
So I guess this here could get closed as currently by design?

**Comment by Jojo-Schmitz** [comment]
Or get moved in some 'hold-third-party' state?

**Comment by rettinghaus** [comment]
I've added handling of percussion clefs in #22545, but so far that's all that can be done here.
